.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,66 +1,32 @@
-<!--
-  ☝️How to write a good PR title:
-  - Prefix it with [ComponentName] (if applicable), for example: [Button]
-  - Start with a verb, for example: Add, Delete, Improve, Fix…
-  - Give as much context as necessary and as little as possible
-  - Prefix it with [WIP] while it’s a work in progress
--->
+<!-- PR title: [ComponentName] *Verb*ing a ... -->
 
 ### WHY are these changes introduced?
 
-Fixes #0000 <!-- link to issue if one exists -->
-
-<!--
-  Context about the problem that’s being addressed.
--->
+Fixes #0000
 
 ### WHAT is this pull request doing?
 
-<!--
-  Summary of the changes committed.
+Before / after screenshots are appreciated for UI changes.
+Include alt text that describes the screenshot.
+Put any gifs inside a `<details>` block.
```

Comment:
Yeah, I did that on purpose. I was thinking that If it's not meant to be seen anyway then it shouldn't matter if it's a comment or some text that needs to be replaced. Personally I delete the comments when I'm making a new PR, so making it text just makes it shorter since you don't have the tags in there as well